src/app/components/Header/Header.module.scss

```
@@ -9,3 +9,42 @@
     font-size: 13px;
   }
 }
+
+.homepage-hero {
+  position: relative;
+  overflow: hidden;
+
+  &::before {
+    content: '';
+    position: absolute;
+    top: -20%;
+    left: -10%;
+    right: -10%;
+    bottom: 0;
+    background: linear-gradient(
+        90deg,
+        rgba(76, 43, 100, 0.85) 12%,
+        rgba(0, 94, 162, 0.85) 50%,
+        rgba(0, 129, 161, 0.85) 98%
+      ),
+      url('/images/homepage-hero-bg.jpeg');
+    background-size: cover;
+    background-position: center 40%;
+    transform: scale(1.2);
+    z-index: 0;
+  }
+
+  .usa-nav-container,
+  .usa-header,
+  .usa-logo,
+  .usa-navbar {
+    position: relative;
+    z-index: 1;
+  }
+  section h1,
+  section p {
+    position: relative;
+    z-index: 1;
+    color: white !important;
```

I believe we can modify this since this styling appears to take precedence.
```suggestion
    color: white;
```

---

done